packages/edge-gateway/src/utils/deny-list.js

```diff
@@ -1,13 +0,0 @@
-import * as uint8arrays from 'uint8arrays'
```

Comment:
All good here, but don't forget we need to update file `.github/workflows/cron-denylist.yml` for new path. Also docs https://github.com/web3-storage/reads/tree/edge-gateway-v1.0.4/packages/edge-gateway#deny-list should be moved to new package readme